### PR TITLE
Update amazons3 connectors for VS code extension

### PR DESCRIPTION
**Pull request by DedunuKarunarathne**
## Purpose
This PR updates the following pages.
- Amazon S3 Connector Example - 2.x
- AmazonSQS Connector Example
- Amazon Lambda Connector Example

### CODE REVIEW

File: en/docs/reference/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 1]
```diff
@@ -234,18 +238,18 @@ You can further refer the application deployed through the CLI tool. See the ins
...
-6. Log in to the AWS Management Console.
+3. Log in to the AWS Management Console.
 
-7. Navigate to the AWS Lambda and Functions tab.
+4. Navigate to the AWS Lambda and Functions tab.
    <img src="{{base_path}}/assets/img/integrate/connectors/awslambdafunction.png" title="Amazon Lambda Function" width="800" alt="Amazon Lambda Function"/>
    
-8. Next you need to execute the function. Navigate to **Configure test events**. <br>
+5. Next you need to execute the function. Navigate to **Configure test events**. <br>
    <img src="{{base_path}}/assets/img/integrate/connectors/configuretestevent.png" title="Configure Test Event" width="800" alt="Configure Test Event"/>
```
Comment (chanikag) [inline en/docs/reference/connectors/amazonlambda-connector/amazonlambda-connector-example.md:247]: Please verify if the image is properly aligned. 

 
Screenshot 2024-07-19 at 2 22 50 PM

File: en/docs/reference/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 2]
```diff
@@ -94,17 +111,14 @@ Follow these steps to set up the Integration Project and the Connector Exporter
...
             <faultSequence/>
         </resource>
     </api> 
     ```
-3. Now we can export the imported connector and the API into a single CAR application. The CAR application is what we are going to deploy during server runtime.
-
-{!includes/reference/connectors/exporting-artifacts.md!}
+5. Now we can export the imported connector and the API into a single CAR application. The CAR application is what we are going to deploy during server runtime.
 
 ## Create Amazon Lambda Deployment Package (Lambda function)
```
Comment (chanikag) [inline en/docs/reference/connectors/amazonlambda-connector/amazonlambda-connector-example.md:120]:
```suggestion
## Create Amazon Lambda deployment package (Lambda function) 
```

File: en/docs/reference/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 3]
```diff
@@ -124,14 +138,14 @@ In this scenario we created sample AWS Deployment Package (Lambda function) in P
...
-Please use command line terminal or shell to run following commands. Commands are shown in listings preceded by a prompt symbol ($) and the name of the current directory, when appropriate:
+Please use the command line terminal or shell to run the following commands. Commands are shown in listings preceded by a prompt symbol ($) and the name of the current directory, when appropriate:
 
 ```
 ~/Documents$ zip addingNumbers.zip addingNumbers.py
  adding: addingNumbers.py.py (deflated 17%)
 ```
 
-## Upload Amazon Lambda Deployment Package (ZIP archive) in to the AWS S3 bucket
+## Upload Amazon Lambda Deployment Package (ZIP archive) into the AWS S3 bucket
```
Comment (chanikag) [inline en/docs/reference/connectors/amazonlambda-connector/amazonlambda-connector-example.md:148]:
```suggestion
## Upload Amazon Lambda deployment package (ZIP archive) into the AWS S3 bucket
```

File: en/docs/reference/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 4]
```diff
@@ -155,13 +169,13 @@ You can download the ZIP file and extract the contents to get the project code.
 
 ## Deployment
 
-Follow these steps to deploy the exported CApp in the integration runtime. 
+Follow these steps to deploy the exported Carbon Application (CApp) in the integration runtime. For more info on deploying artifacts, refer to the [Deploying Artifacts]({{base_path}}/develop/deploy-artifacts.md).
```
Comment (chanikag) [inline en/docs/reference/connectors/amazonlambda-connector/amazonlambda-connector-example.md:172]:
```suggestion
Follow these steps to deploy the exported Composite Application (CApp) in the integration runtime. For more info on deploying artifacts, refer to the [Deploying Artifacts]({{base_path}}/develop/deploy-artifacts.md).
```

File: en/docs/reference/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 5]
```diff
@@ -172,17 +186,7 @@ You can further refer the application deployed through the CLI tool. See the ins
...
-    ./mi remote login
-    ```
-2. Provide default credentials admin for both username and password.
-
-3. In order to view the proxy services deployed, execute the following command.
-    ```
-    ./mi api show
-    ```
-4. Send a POST request using a CURL command or sample client.
+1. Invoke the API using a CURL command or sample client.
```
Comment (chanikag) [inline en/docs/reference/connectors/amazonlambda-connector/amazonlambda-connector-example.md:189]:
```suggestion
1. Invoke the API using a cURL command or a sample client.
```

File: en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md [hunk 6]
```diff
@@ -6,17 +6,17 @@ The AmazonS3 Connector allows you to access the Amazon Simple Storage Service (A
...
 
 All three operations are exposed via an API. The API with the context `/s3connector` has three resources:
 
 * `/createbucket` - Once invoked, it will create a bucket in Amazon with the specified name
-* `/addobject`  - The incoming message will be stored into the specified bucket with the specified name
+* `/addobject`  - The incoming message will be stored in the specified bucket with the specified name
 * `/info` - Once invoked, it will read the specified file from the specified bucket and respond with the content of the file
 
-Following diagram shows the overall solution. The user creates a bucket, stores some message into the bucket, and then receives it back.
+The following diagram shows the overall solution. The user creates a bucket, stores some message in the bucket, and then receives it back.
```
Comment (chanikag) [inline en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md:19]:
```suggestion
The following diagram shows the overall solution. The user creates a bucket, stores some messages in the bucket, and then receives it back.
```

File: en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md [hunk 7]
```diff
@@ -26,34 +26,41 @@ If you do not want to configure this yourself, you can simply [get the project](
 
 ## Setting up the environment
```
Comment (chanikag) [inline en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md:27]:
```suggestion
## Set up the environment
```

File: en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md [hunk 8]
```diff
@@ -26,34 +26,41 @@ If you do not want to configure this yourself, you can simply [get the project](
...
 
-Please follow the steps mentioned at [Setting up Amazon S3]({{base_path}}/reference/connectors/amazons3-connector/amazons3-connector-config) document in order to create a Amazon S3 account and obtain credentials you need to access the Amazon APIs. Keep them saved to be used in the next steps.
+Please follow the steps mentioned at [Setting up Amazon S3]({{base_path}}/reference/connectors/amazons3-connector/amazons3-connector-config) document in order to create an Amazon S3 account and obtain the credentials you need to access the Amazon APIs. Keep them saved to be used in the next steps.
 
-## Configure the connector in WSO2 Integration Studio
+## Setup the Integration Project
```
Comment (chanikag) [inline en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md:31]:
```suggestion
## Set up the integration project
```

File: en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md [hunk 9]
```diff
@@ -26,34 +26,41 @@ If you do not want to configure this yourself, you can simply [get the project](
...
 
-Follow these steps to set up the Integration Project and import AmazonS3 connector into it.
-
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!}
+{!includes/build-and-run.md!}
 
 ## Creating the Integration Logic
```
Comment (chanikag) [inline en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md:35]:
```suggestion
## Create the integration logic
```

File: en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md [hunk 10]
```diff
@@ -63,48 +70,70 @@ Follow these steps to set up the Integration Project and import AmazonS3 connect
...
         2. The [IAM role for authentication](https://docs.amazonaws.cn/en_us/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html) is available only with Amazon S3 connector v2.0.2 and above.
 
-    <img src="{{base_path}}/assets/img/integrate/connectors/s3-connector-2x/s3-connector-connection.jpg" title="Configuring a new connection" width="500" alt="Amazon S3 use case"/>
+    <img src="{{base_path}}/assets/img/integrate/connectors/s3-connector-2x/s3-connector-create-new-connection.png" title="Configuring a new connection" width="500" alt="Configuring a new connection"/>
 
-5. After the connection is successfully created, select the created connection as 'Connection' from the drop down menu in the properties window.
+7. After the connection is successfully created, select the created connection as **Amazon S3Connection** from the dropdown menu in the **Add createBucket** form.
 
-6. Next, configure the following parameters in the properties window,
+8. Next, configure the following parameters:
```
Comment (chanikag) [inline en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md:77]:
```suggestion
8. Next, configure the following parameters.
```

File: en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md [hunk 11]
```diff
@@ -140,30 +167,39 @@ Follow these steps to set up the Integration Project and import AmazonS3 connect
...
 * As `awsSecretAccessKey` use the secret key obtained from Amazon S3 setup and update the above API configuration.
 * Note that `region`, `connectionName` and credentials are hard coded. Please change them as per the requirement.
 * For more information please refer the [reference guide]({{base_path}}/reference/connectors/amazons3-connector/amazons3-connector-reference) for Amazon S3 connector.
 
-Now we can export the imported connector and the API into a single CAR application. CAR application is the one we are going to deploy to server runtime.
-
-{!includes/reference/connectors/exporting-artifacts.md!}
-
-Now the exported CApp can be deployed in the integration runtime so that we can run it and test.
+Now we can export the artifacts into a Carbon Application (CApp) and deploy it to the server runtime to run it and test it.
```
Comment (chanikag) [inline en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md:196]:
```suggestion
Now we can export the artifacts into a composite application (CApp) and deploy it to the server runtime to run it and test it.
```

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md [hunk 12]
```diff
@@ -14,11 +14,11 @@ If you do not want to configure this yourself, you can simply [get the project](
 
 ## Setting up the environment
```
Comment (chanikag) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md:15]:
```suggestion
## Set up the environment 
```

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md [hunk 13]
```diff
@@ -14,11 +14,11 @@ If you do not want to configure this yourself, you can simply [get the project](
...
+1. Please follow the steps mentioned in the [Setting up the Amazon S3 Environment ]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document in order to create an Amazon account and obtain access key id and secret access key. Keep them saved to be used in the next steps.  
 
 2. In this example we will be using XPath 2.0 which needs to be enabled in the product as shown below before starting the integration service. 
 
-    If you are using the Micro Integrator of **EI 7** or **APIM 4.0.0**, you need to enable this property by adding the following to the PRODUCT-HOME/conf/deployment.toml file. You can further refer to the [Product Configurations](https://apim.docs.wso2.com/en/4.2.0/reference/config-catalog/#http-transport).
+    If you are using the Micro Integrator of **EI 7** or **MI 4.x.0**, you need to enable this property by adding the following to the PRODUCT-HOME/conf/deployment.toml file. You can further refer to the [Product Configurations](https://apim.docs.wso2.com/en/4.2.0/reference/config-catalog/#http-transport).
```
Comment (chanikag) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md:21]: It should be MI 4.0.0. This is referring to older versions.

Comment (chanikag) [reply en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md:21]: Let's point to the config catalog content in this doc space

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md [hunk 14]
```diff
@@ -29,16 +29,71 @@ If you do not want to configure this yourself, you can simply [get the project](
...
 3. In this example we use the SimpleStockQuote service backend. Therefore, the SimpleStockQuote service needs to be started. 
 
-## Configure the connector in WSO2 Integration Studio
+## Set up the Integration Project
 
-Follow these steps to set up the Integration Project and the Connector Exporter Project. 
+{!includes/build-and-run.md!}
 
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!} 
+## Creating the Integration Logic
```
Comment (chanikag) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md:36]:
```suggestion
## Create the integration logic
```

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md [hunk 15]
```diff
@@ -29,16 +29,71 @@ If you do not want to configure this yourself, you can simply [get the project](
...
 
-1. First let's create the following sequences, which are buildMessage, createQueue, sendMessage and ReceiveAndForwardMessage. Right click on the created Integration Project and select, -> **New** -> **Sequence** to create the Sequence. 
-    <img src="{{base_path}}/assets/img/integrate/connectors/add-sequence.jpg" title="Adding a Sequence" width="800" alt="Adding a Sequence"/>
+1. First let's create a connection to the Amazon SQS instance. Navigate to **MI Project Explorer** > **Local Entries** > **Connections** and click on the **+** sign next to **Connections** to open the **Add New Connection** view.
+2. Select the **Amazonsqs** connector.
+   Configure the below values.
 
-2. Provide the Sequence name as buildMessage. You can go to the source view of the XML configuration file of the API and copy the following configuration. In this sequence we are taking the user's input `companyName` and we build the message using a Payload Factory Mediator. 
+       - Connection Name - Unique name to identify the connection by.
```
Comment (chanikag) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md:42]:
```suggestion
       - Connection Name - A unique name to identify the connection by.
```

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md [hunk 16]
```diff
@@ -29,16 +29,71 @@ If you do not want to configure this yourself, you can simply [get the project](
...
+           </markForSuspension>
+       </address>
+   </endpoint>
+   ```
+6. Then create the following sequences, which are buildMessage, createQueue, sendMessage and ReceiveAndForwardMessage.
+7. Navigate to **MI Project Explorer** > **Sequences** and click on the **+** sign next to Sequences to open the **Create New Sequence** form.
+    
+    <img src="{{base_path}}/assets/img/integrate/connectors/amazon-sqs/create-new-sequence.png" title="Adding a Sequence" width="800" alt="Adding a Sequence"/>
+
+8. Provide the Sequence name as `buildMessage` and click **Create**. You can go to the source view of the XML configuration file of the sequence and copy the following configuration. In this sequence we are taking the user's input `companyName` and we build the message using a Payload Factory Mediator.
```
Comment (chanikag) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md:96]:
```suggestion
8. Provide the sequence name as `buildMessage` and click **Create**. You can go to the source view of the XML configuration file of the sequence and copy the following configuration. In this sequence we are taking the user's input `companyName` and we build the message using a Payload Factory Mediator. 
```

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md [hunk 17]
```diff
@@ -86,35 +135,23 @@ Follow these steps to set up the Integration Project and the Connector Exporter
...
+        <amazonsqs.sendMessage configKey="AMAZON_SQS_CONNECTION">
+            <queueId>{$ctx:queueId}</queueId>
+            <queueName>{$ctx:queueName}</queueName>
+            <messageBody>{$ctx:target_property}</messageBody>
+        </amazonsqs.sendMessage>
+    </sequence>
+  ```
 
-  5. Create the ReceiveAndForwardMessage sequence as shown below. In this sequence, we will receive the message from the Amazon SQS queue and forward it into the StockQuote Endpoint. 
+5. Create the ReceiveAndForwardMessage sequence as shown below. In this sequence, we will receive the message from the Amazon SQS queue and forward it to the StockQuote Endpoint.
```
Comment (chanikag) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md:150]:
```suggestion
5. Create the `ReceiveAndForwardMessage` sequence as shown below. In this sequence, we will receive the message from the Amazon SQS queue and forward it to the StockQuote Endpoint. 
```

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md [hunk 18]
```diff
@@ -137,34 +174,33 @@ Follow these steps to set up the Integration Project and the Connector Exporter
...
+                <sequence key="ReceiveAndForwardMessage"/>
+                <respond/>
+            </inSequence>
+            <faultSequence/>
+        </resource>
+    </api>
     ```
 
-{!includes/reference/connectors/exporting-artifacts.md!}
+Now we can export the artifacts into a Carbon Application (CApp) and deploy it to the server runtime to run it and test it.
```
Comment (chanikag) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md:197]:
```suggestion
Now we can export the artifacts into a composite application (CApp) and deploy it to the server runtime to run it and test it.
```

File: en/docs/reference/connectors/redis-connector/redis-connector-example.md [hunk 19]
```diff
@@ -7,16 +7,16 @@ The Redis connector allows you to access the Redis commands from an integration
 Given below is a sample scenario that demonstrates how to work with the WSO2 Redis Connector and access the Redis server, using Redis commands.
 
 The user sends the request to invoke an API to get stock details. This REST call will get converted into a SOAP message and is sent to the back-end service. While the response from the backend service is converted back to JSON and sent back to the API caller, the integration runtime will extract stock volume details from the response and store it into a configured Redis server.
```
Comment (chanikag) [inline en/docs/reference/connectors/redis-connector/redis-connector-example.md:9]:
```suggestion
The user sends the request to invoke an API to get stock details. This REST request will get converted into a SOAP message and is sent to the back-end service. While the response from the backend service is converted back to JSON and sent back to the API caller, the integration runtime will extract stock volume details from the response and store it into a configured Redis server.
```

File: en/docs/reference/connectors/redis-connector/redis-connector-example.md [hunk 20]
```diff
@@ -485,15 +485,15 @@ You can download the ZIP file and extract the contents to get the project code.
 
 ## Deployment
 
-Follow these steps to deploy the exported CApp in the integration runtime. 
+Follow these steps to deploy the exported Carbon Application (CApp) in the integration runtime.
```
Comment (chanikag) [inline en/docs/reference/connectors/redis-connector/redis-connector-example.md:488]:
```suggestion
Follow these steps to deploy the exported composite application (CApp) in the integration runtime. 
```

File: en/docs/reference/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 1 above)
Comment (DedunuKarunarathne) [reply en/docs/reference/connectors/amazonlambda-connector/amazonlambda-connector-example.md:247]: This is already aligned
Screenshot 2024-07-22 at 10 55 06

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md (hunk 13 above)
Comment (DedunuKarunarathne) [reply en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md]: Fixed with dc82fd313f75013cb3fc9b1578a14b94ad564545

**Review by chanikag — APPROVED** [review]

**Pull request merged**